### PR TITLE
Added get messages method

### LINKED ISSUE #91: [view messages] Add get_messages method
It will get the messages data from DB and display it in view-messages page in chronological order.
Use Messages table.

### PR DESCRIPTION
**Pull request by morbitton**
Fixes #91 

@lioramilbaum @Yarinh55 
Hi, it's the base , because the database didn't merge yet

### CODE REVIEW

**Comment by lmilbaum** [comment]
@morbitton db PRs are merged. Go ahead and verify your code. Add tests to travis please.

**Review by lmilbaum — CHANGES_REQUESTED** [review]
pull latest changes from upstream, rebase
add tests to travis

File: connect.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import mysql.connector
+
```
Comment (AviadP) [inline connect.py]: you must use functions here. without it you file will not be useful to others

File: connect.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import mysql.connector
+
+mydb = mysql.connector.connect(
+    host="localhost",
+    user="root",
+    passwd="",
+    database="test"
+)
+
+print(mydb)
```
Comment (AviadP) [inline connect.py]: debug printing shell not be pushed to production

Comment (AviadP) [inline connect.py]: also, python have easy to use log module, which is much better for debugging purposes (in this case, you can leave it in production with proper printing level - Debug, Error, Info etc.)

**Review by AviadP — CHANGES_REQUESTED** [review]

**Comment by lmilbaum** [comment]
@morbitton what is the status of this PR?

**Comment by morbitton** [comment]
need to fix the problem with travis and upload

**Comment by lmilbaum** [comment]
OK. Awaiting for those changes. LMK if you are blocked or not.

**Comment by morbitton** [comment]
closed, open new one

**Pull request closed without merging**